pkg/controller/directimagemigration/directimagemigration_controller.go

```diff
@@ -130,7 +124,7 @@ func (r *ReconcileDirectImageMigration) Reconcile(request reconcile.Request) (re
 	// Set MigMigration name key on logger
 	migration, err := imageMigration.GetMigrationForDIM(r)
 	if migration != nil {
-		log.SetValues("migMigration", migration.Name)
+		log.Real = log.WithValues("migMigration", migration.Name)
```

Comment:
@sseago what is log.Real? Is this the logger we're supposed to use throughout the codebase to make sure we get these keys?

Reply:
log.Real is the internal logger that log deals with. In the latest version of the konveyor/controller log package, `SetValues` doesn't exist anymore. `WithValues` does the same thing, but unfortunately the return type requires me to assign it to `log.Real` instead of `log`. In consulting with @jortel though, this is equivalent to what the old code did.